### ISSUE TITLE
IronA11yAnnouncer.requestAvailability static method missing from TypeScript .d.ts file.

### ISSUE THREAD

**Issue by freshp86** [PolymerElements/iron-a11y-announcer#52]
`iron-a11y-announcer.d.ts` is missing an entry for the static `IronA11yAnnouncer.requestAvailability()` method. This was discovered while investigating using TypeScript in Chromium's WebUI codebase, and we had to add this manually for now with the following diff

```diff
diff --git a/third_party/polymer/v3_0/components-chromium/iron-a11y-announcer/iron-a11y-announcer.d.ts b/third_party/polymer/v3_0/components-chromium/iron-a11y-announcer/iron-a11y-announcer.d.ts
index b60ac3ba18a0..1646ce15f0f0 100644
--- a/third_party/polymer/v3_0/components-chromium/iron-a11y-announcer/iron-a11y-announcer.d.ts
+++ b/third_party/polymer/v3_0/components-chromium/iron-a11y-announcer/iron-a11y-announcer.d.ts
@@ -34,6 +34,7 @@ declare class IronA11yAnnouncer {
    */
   announce(text: string): void;
   _onIronAnnounce(event: any): void;
+  static requestAvailability(): void;
 }
 
 interface IronA11yAnnouncer extends LegacyElementMixin, HTMLElement {
```

@usergenic 
I believe this file is generated automatically at [1], and therefore possibly a bug in the `gen-typescript-declarations` tool itself?

[1] https://github.com/PolymerElements/iron-a11y-announcer/blob/master/package.json#L27

**Comment by usergenic** [comment]
Hey @freshp86 --

Apparently the gen-typescript-declarations tool isn't really maintained and so it's going to be better to update these declarations manually from now on.  I'm looking into making a PR to remove the script from the package.json and submit your change to the .d.ts as part of the source.

**Comment by freshp86** [comment]
Thanks for the response, that makes sense. Note that the same tool is used throughout many iron and paper elements.

Also, it would help if a new NPM version could be rolled after this fix, so that we can remove the local patch we added in the Chromium repository at https://source.chromium.org/chromium/chromium/src/+/master:third_party/polymer/v3_0/chromium_dts.patch;l=1-14

**Comment by usergenic** [comment]
@freshp86 working on it.  There's some issues with the npm package `wd` not getting modules built during post-install scripts owing to newer version of npm not running said scripts.  Attempting to get that fixed on our travis configs so we can get green tests and merge the PR.

**Comment by usergenic** [comment]
PR for tracking https://github.com/PolymerElements/iron-a11y-announcer/pull/53

**Comment by freshp86** [comment]
Thank you for fixing! The CL to include the new 3.2.0 version in the Chromium repository is at https://chromium-review.googlesource.com/c/chromium/src/+/2866426.
